data/starts.txt

```diff
@@ -0,0 +1,227 @@
+	date 18 6 3020
+	to display
+		or
+			has "main plot completed"
+			has "Ask the Hai about the Unfettered"
+	on display
+		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
+		thumbnail ""
+		date "middle of 3020"
+	to reveal
+		has "main plot completed"
+		has "Ask the Hai about the Unfettered"
+	on reveal
+		name "Hai Start"
+		description "There could be interesting possibilities up north, once all crisies are resolved."
+		description "You could have started a different way, went a different path."
+		date "Jun 3020"
+		credits 1.431M
+		debt 1.431M
+	to unlock
+		has "main plot completed"
+		has "event: hai-human resolution announced"
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "hai start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+	set "skip FW"
+	set "skip HR"
+
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and video you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
+	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
+	scene "thumbnail/hai aphid"
+	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
+	`	You have, after all, heard tales of pirates and quick money... And up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them the strength and cunning they respect.`
+	choice
+		`	(Go to the bank.)`
+	scene "scene/hai bank"
+	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
+	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some part of the ship in order to be able to afford flying it!`
+	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`
+	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
```

Comment:
> 1 million 431 thousand

This hurts my eyes

Reply:
```suggestion
	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1,431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
```
?